packages/web/pages/_app.tsx

```diff
@@ -23,31 +23,48 @@ import {
   PromotedLBPPoolIds,
 } from "../config";
 import { useAmplitudeAnalytics } from "../hooks/use-amplitude-analytics";
+import {
+  setDefaultLanguage,
+  setTranslations,
+  useTranslation,
+} from "react-multi-lang";
+
+import en from "../localizations/en.json";
+import "../localizations/dayjs-locale-en.js";
+import fr from "../localizations/fr.json";
```

Comment:
Can we try to use next/dynamic or something to import these as needed, instead of including it in the main bundle? It doesn't appear to be very large right now, but I'm more concerned about when we add more text and languages. IMO we should not have any other translations in the browser except the ones the user needs. May need to wrap it in a hook or something.